content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -52,10 +52,12 @@ The Kubelet acts as a bridge between the Kubernetes master and the nodes. It man
 
 cAdvisor is an open source container resource usage and performance analysis agent. It is purpose-built for containers and supports Docker containers natively. In Kubernetes, cAdvisor is integrated into the Kubelet binary. cAdvisor auto-discovers all containers in the machine and collects CPU, memory, filesystem, and network usage statistics. cAdvisor also provides the overall machine usage by analyzing the 'root' container on the machine.
 
-On most Kubernetes clusters, cAdvisor exposes a simple UI for on-machine containers on port 4194. Here is a snapshot of part of cAdvisor's UI that shows the overall machine usage:
+In Kubernetes version 1.9 and earlier, cAdvisor exposes a simple UI for on-machine containers on default port 4194. Here is a snapshot of part of cAdvisor's UI that shows the overall machine usage:
 
 ![cAdvisor](/images/docs/cadvisor.png)
 
+In version 1.10 and later, the Advisor module does not provide this UI. Instead, you can [run cAdvisor as DaemonSet](https://github.com/google/cadvisor/tree/master/deploy/kubernetes).
```

Comment:
See also line 55. The original is unclear (before 1.10 and after 1.10 -- but what about IN 1.10?). Are lines 55 and 59 now correct? They are clearer, but the original could also have meant that the UI was available in 1.10 but not in 1.11. Please specify either in new commit or in comment. Thanks!